x/gov/keeper/keeper.go

```diff
@@ -89,7 +86,7 @@ func (k Keeper) GetAuthority() string {
 // CONTRACT: the parameter Subspace must have the param key table already initialized
 func NewKeeper(
 	cdc codec.Codec, storeService corestoretypes.KVStoreService, authKeeper types.AccountKeeper,
-	bankKeeper types.BankKeeper, sk types.StakingKeeper, distrKeeper types.DistributionKeeper,
+	bankKeeper types.BankKeeper, calculateVoteResultsAndVotingPowerFn CalculateVoteResultsAndVotingPowerFn, distrKeeper types.DistributionKeeper,
 	router baseapp.MessageRouter, config types.Config, authority string, initOptions ...InitOption,
```

Comment:
Nits:

- it might make sense for the TallyFn to be an arugment after all keepers (so just moving after `distrKeeper`
- In 53, we added this `initOptions` variadic argument, but the only usage of it was to provide a tallyFn to override the default tally Fn - might make sense to remove this now that are just having the fn as an explicit argument

Reply:
fixed: https://github.com/cosmos/cosmos-sdk/pull/25615/commits/90775e5086e8e2bbdeedfc2bfb7375863670c01e